hack/make-docs.sh

```diff
@@ -35,6 +35,6 @@ mkdir -p site
 # Generate docs with mkdocs
 mkdocs build
 # Generate v1alpha2 API docs
-./hack/api-docs/generate.sh v1alpha2 site/v1alpha2-spec.html
+./hack/api-docs/generate.sh site/spec.html
 # Add them to spec page originally generated by mkdocs
-run::sed -e '/REPLACE_WITH_GENERATED_CONTENT/{r site/v1alpha2-spec.html' -e 'd;}' site/v1alpha2/references/spec/index.html
```

Comment:
For the sake of the old link, it would be nice to leave this line in place:
https://deploy-preview-1238--kubernetes-sigs-gateway-api.netlify.app/v1alpha2/references/spec/

Reply:
Good catch - will fix